charts/incubator/lldap/values.yaml

```
@@ -0,0 +1,69 @@
+image:
+  repository: nitnelave/lldap
+  pullPolicy: IfNotPresent
+  tag: latest-debian
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+
+service:
+  main:
+    ports:
+      main:
+        targetPort: 17170
+        port: 17170
+  ldap:
+    ports:
+      main:
+        targetPort: 3890
+        port: 3890
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          command: ["/app/lldap"]
+          args: ["run"]
+          env:
+            LLDAP_LDAP_BASE_DN: "dc=example,dc=com"
+            LLDAP_LDAP_USER_PASS: "change me"
+            LLDAP_JWT_SECRET:
+              - secretRef:
+                  name: secrets
+                  key: LLDAP_JWT_SECRET
+            LLDAP_key_file: "/data/private_key"
+            LLDAP_database_url:
+              secretKeyRef:
+                name: cnpg-main-urls
+                key: std
+
+database:
+  # -- Database type,
+  # must be one of: 'sqlite', 'mysql' or 'postgresql'.
+  type: postgresql
+  # -- Enable DB Write-Ahead-Log for SQLite,
+  # disabled for other databases. https://github.com/dani-garcia/bitwarden_rs/wiki/Running-without-WAL-enabled
+  wal: false
+  ## URL for external databases (mysql://user:pass@host:port or postgresql://user:pass@host:port).
+  # url: ""
+  ## Set the size of the database connection pool.
+  # maxConnections: 10
+  ## Connection retries during startup, 0 for infinite. 1 second between retries.
+  # retries: 15
```

```suggestion
database:
  # -- Database type,
  # must be one of: 'sqlite', 'mysql' or 'postgresql'.
  type: postgresql
  # -- Enable DB Write-Ahead-Log for SQLite,
  # disabled for other databases. https://github.com/dani-garcia/bitwarden_rs/wiki/Running-without-WAL-enabled
  wal: false
  ## URL for external databases (mysql://user:pass@host:port or postgresql://user:pass@host:port).
  # url: ""
  ## Set the size of the database connection pool.
  # maxConnections: 10
  ## Connection retries during startup, 0 for infinite. 1 second between retries.
  # retries: 15
```

What the duck is this? You didn't connect this anywhere it looks like....

---

Originally copied from the Vaultwarden chart. Intended originally to offer the choice to switch databases when it was hazy whether postgres was supported then was unsure how to simplify it down. Didn't connect it up as I reversed my original intention as only offering postgres is better a route?

---

Lood again and added a commit to fix this.